testdata/fuzz/invariant/InvariantInnerContract.t.sol

```diff
@@ -1,14 +1,14 @@
 // SPDX-License-Identifier: Unlicense
 pragma solidity >=0.8.0;
 
-import "ds-test/test.sol";
+import "ds-test/test.sol"; /////////////////////////////////////////////////////////////
```

Comment:
what happened here? :D  maybe a forge-fmt bug? @rkrasiuk @jpopesculian

Reply:
@jpopesculian postfix comment found the nearest source unit anchor?

Reply:
it does look like a bug to me

Reply:
yeah, `forge fmt`

Reply:
extracted to issue #2713 ✔️